linode/manager
# change: [DI-22390] - Enhanced monitor page for contextual view

## Description 📝

Enhanced the monitor page for contextual view as per the latest figma https://www.figma.com/design/uvdOz9Xs0RVdMRJMCIgS9H/ACLP---Contextual-views?node-id=12209-52065&t=4mG9hXqZrRpancpc-0

## Changes  🔄

List any change(s) relevant to the reviewer.

1. Updated layouts in CloudPulseDashboardWithFilters.tsx component
2. Added Dashboard select dropdown with in disabled state with default value selected.
3. Added a getFilters method in FilterBuilder.ts file


## Target release date 🗓️

20th Dec 2024

## Preview 📷

**Include a screenshot or screen recording of the change.**

:lock: Use the [Mask Sensitive Data](https://cloud.linode.com/profile/settings) setting for security.

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
|![Screenshot 2024-12-10 at 12 35 33 PM](https://github.com/user-attachments/assets/e3825476-f8ca-4f95-920e-a9743b806276)|![Screenshot 2024-12-16 at 12 45 17 PM](https://github.com/user-attachments/assets/ffbacf64-c223-4d03-bbd0-10b82d2f3cf3)|


## How to test 🧪

### Prerequisites

(How to setup test environment)

1. Go to databases ( if no aiven database available in your profile then switch to mock)
2. Select any database instance
3. Select the monitor.
4. You can verify the changes compare to alpha env.

<details>
<summary> Author Checklists </summary>

## As an Author, to speed up the review process, I considered 🤔

👀 Doing a self review
❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
🤏 Splitting feature into small PRs
➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
🧪 Providing/improving test coverage
 🔐 Removing all sensitive information from the code and PR description
🚩 Using a feature flag to protect the release
👣 Providing comprehensive reproduction steps
📑 Providing or updating our documentation
🕛 Scheduling a pair reviewing session
📱 Providing mobile support
♿  Providing accessibility support

<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅


</details>
